tmail-backend/combined-identity/src/test/java/com/linagora/tmail/combined/identity/CombinedUsersRepositoryTest.java

```diff
@@ -0,0 +1,128 @@
+package com.linagora.tmail.combined.identity;
+
+import static org.apache.james.user.ldap.DockerLdapSingleton.ADMIN;
+import static org.apache.james.user.ldap.DockerLdapSingleton.ADMIN_LOCAL_PART;
+import static org.apache.james.user.ldap.DockerLdapSingleton.ADMIN_PASSWORD;
+import static org.apache.james.user.ldap.DockerLdapSingleton.DOMAIN;
+
+import org.apache.commons.configuration2.BaseHierarchicalConfiguration;
+import org.apache.commons.configuration2.HierarchicalConfiguration;
+import org.apache.commons.configuration2.plist.PropertyListConfiguration;
+import org.apache.commons.configuration2.tree.ImmutableNode;
+import org.apache.james.backends.cassandra.CassandraClusterExtension;
+import org.apache.james.domainlist.api.DomainList;
+import org.apache.james.user.cassandra.CassandraUsersDAO;
+import org.apache.james.user.cassandra.CassandraUsersRepositoryModule;
+import org.apache.james.user.ldap.LdapGenericContainer;
+import org.apache.james.user.ldap.ReadOnlyUsersLDAPRepository;
+import org.apache.james.user.lib.UsersRepositoryContract;
+import org.apache.james.user.lib.UsersRepositoryImpl;
+import org.apache.james.user.lib.model.Algorithm;
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.extension.RegisterExtension;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class CombinedUsersRepositoryTest implements CombinedUsersRepositoryContract {
+
+    static final Logger LOGGER = LoggerFactory.getLogger(CombinedUsersRepositoryTest.class);
+
+    static LdapGenericContainer ldapContainer = LdapGenericContainer.builder()
+        .domain(DOMAIN)
+        .password(ADMIN_PASSWORD)
+        .build();
```

Comment:
Why not using `DockerLdapSingleton.ldapContainer` that is the exact same declaration?